app/src/main/java/com/github/se/wanderpals/ui/screens/suggestion/SuggestionDetail.kt

```diff
@@ -76,10 +76,7 @@ fun SuggestionDetail(
         topBar = {
           TopAppBar(
               title = {
-                Text(
-                    text = suggestion.stop.title,
-                    style = MaterialTheme.typography.titleLarge,
-                    modifier = Modifier.testTag("SuggestionTitle"))
+                Text(text = suggestion.stop.title, modifier = Modifier.testTag("SuggestionTitle"))
```

Comment:
It looks good too without the `style = MaterialTheme.typography.titleLarge`. In the future, we can store small changes to make, and make an issue with lots of small changes.